src/app/resources/resources.component.ts

```diff
@@ -193,4 +193,10 @@ export class ResourcesComponent implements OnInit, AfterViewInit {
     this.router.navigate([ '/' ]);
   }
 
+  download(dataFile) {
+    const a = document.createElement('a');
```

Comment:
We should not directly modify the DOM unless absolutely necessary.  Angular should handle all DOM manipulation otherwise we may see unintended side effects.